src/network/networkd-address.c

```diff
@@ -448,6 +448,127 @@ static int address_set_masquerade(Address *address, bool add) {
         return 0;
 }
 
+static int netlabel_op(bool add, const char *label, const Address *address) {
```

Comment:
instead of passing a bool as first arg, maybe just pass `uint16_t command` as first argument?

that makes the invocations more self-explanatory (since you imemdiately see if this is about add or about remove, instead of a cryptic "true" or "false), and shortens the code.

Reply:
Nice!